roles/privacyidea/tasks/configure_privacyidea.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Copy uWSGI configuration
+  template:
+    src: privacyidea.ini.j2
+    dest: /etc/uwsgi/apps-available/privacyidea.ini
+  notify: restart uWSGI
+
+- name: Enable uWSGI app for PrivacyIDEA
+  file:
+    src: /etc/uwsgi/apps-available/privacyidea.ini
+    dest: /etc/uwsgi/apps-enabled/privacyidea.ini
+    state: link
+  notify: restart uWSGI
+
+- name: Copy uWSGI service file
+  template:
+    src: uwsgi.service.j2
+    dest: /etc/systemd/system/uwsgi.service
+  notify: restart uWSGI
+
+
+- name: Copy PrivacyIDEA WSGI script
+  template:
+    src: privacyideaapp.py.j2
+    dest: /etc/privacyidea/privacyideaapp.py
+  notify: restart uWSGI
+
+- name: Reload systemd to recognize uwsgi service
+  command: systemctl daemon-reload
+  become: true
+
+- name: Create database tables
+  command: "{{ privacyidea_dir }}/virtualenv/bin/pi-manage create_tables"
+
+- name: Stamping the database to the current database schema
+  command: "{{ privacyidea_dir }}/virtualenv/bin/pi-manage db stamp head -d {{ privacyidea_dir }}/virtualenv/lib/privacyidea/migrations/"
+
+# Creating the new users_service table in MariaDB based on the structure from admin2go
+- name: Create users_service table
+  mysql_query:
+    login_user: "{{ privacyidea_mariadb_username }}"
+    login_password: "{{ privacyidea_mariadb_password }}"
+    login_db: "{{ privacyidea_mariadb_name }}"
+    query: |
+      CREATE TABLE IF NOT EXISTS users_service (
+        id INT AUTO_INCREMENT PRIMARY KEY,
+        username VARCHAR(40) UNIQUE
+      )
+      
+- name: Copy admin SQL resolver configuration file
+  template:
+    src: admin_sql_resolver_config.ini.j2
+    dest: /etc/privacyidea/admin_sql_resolver_config.ini
+    owner: www-data
+    group: www-data
+    mode: '0644'
+  notify: restart uWSGI
+
+- name: Copy User SQL resolver configuration file
+  template:
+    src: user_sql_resolver_config.ini.j2
+    dest: /etc/privacyidea/user_sql_resolver_config.ini
+    owner: www-data
+    group: www-data
+    mode: '0644'
+  notify: restart uWSGI
+
+- name: Set PrivacyIDEA file permissions
+  ansible.builtin.file:
+    path: "{{ item.path }}"
+    owner: www-data
+    group: www-data
+    mode: "{{ item.mode }}"
+  loop:
+    - { path: '/etc/privacyidea/enckey', mode: '0600' }
+    - { path: '/etc/privacyidea/pi.cfg', mode: '0644' }
+    - { path: '/etc/privacyidea/privacyideaapp.py', mode: '0755' }
+    - { path: '/etc/privacyidea/private.pem', mode: '0600' }
+    - { path: '/etc/privacyidea/public.pem', mode: '0644' }
+  become: true
+  notify: restart uWSGI
```

Comment:
same here, are so many restarts of the uWSGI really necessary?

Reply:
You are right, I have deleted it

Reply:
I was wrong, it can stay, as Yannick explained :)